lib/active_remote/typecasting/boolean_typecaster.rb

```diff
@@ -0,0 +1,15 @@
+module ActiveRemote
+  module Typecasting
+    class BooleanTypecaster
+       FALSE_VALUES = ["n", "N", "no", "No", "NO", "false", "False", "FALSE", "off", "Off", "OFF", "f", "F"]
+
+      def self.call(value)
+        case value
+        when *FALSE_VALUES then false
+        when Numeric, /^\-?[0-9]/ then !value.to_f.zero?
```

Comment:
This regex is checking for strings that are only numbers. Its really a poor way of handling this. I think ActiveModel's [boolean typecaster](https://github.com/rails/rails/blob/52ce6ece8c8f74064bb64e0a0b1ddd83092718e1/activemodel/lib/active_model/type/boolean.rb) is much better.